nav2_collision_monitor/src/collision_monitor_node.cpp

```diff
@@ -426,7 +426,7 @@ bool CollisionMonitor::processStopSlowdown(
       robot_action.req_vel.y = 0.0;
       robot_action.req_vel.tw = 0.0;
       return true;
-    } else {  // SLOWDOWN
+    } else if (polygon->getActionType() == SLOWDOWN) {  // SLOWDOWN
```

Comment:
There is no need to keep the comment: it is excplicitly seen which branch we are using from on now.

Reply:
Done